src/__tests__/vue-router.js

```diff
@@ -32,3 +33,17 @@ test('setting initial route', () => {
 
   expect(getByTestId('location-display')).toHaveTextContent('/about')
 })
+
+test('can render with an instantiated Vuex store', async () => {
+  // Instantiate a router with only one route
+  const instantiatedRouter = new VueRouter({
+    routes: [{path: '/special-path', component: Home}],
+  })
+  instantiatedRouter.__isSameInstance = true
```

Comment:
Is this needed for the test to pass? I think that, since tests here serve also the purpose of documentation, I would not showcase this modification as it might confuse people

Reply:
Nope - the test will still pass without it. I added it to truly ensure it's the same instance, but I can remove it.